src/content/utils.ts

```diff
@@ -1,55 +1,32 @@
-export class PaymentEndpointError extends Error {}
-
-export function checkWalletAddressUrlFormat(
-  pointerOrUrl: string,
-  urlOnly = false
-) {
-  const httpUrl = urlOnly
-    ? pointerOrUrl
-    : pointerOrUrl.replace(/^\$/, 'https://')
+export class WalletAddressFormatError extends Error {}
 
+export function checkWalletAddressUrlFormat(walletAddressUrl: string): void {
   let url: URL
   try {
-    url = new URL(httpUrl)
-    if (!url.protocol.match(/https?:/)) {
-      const spec = urlOnly ? '' : 'either a payment pointer or '
-      // noinspection ExceptionCaughtLocallyJS
-      throw new PaymentEndpointError(
-        `SPSP endpoint must be specified as ${spec}fully resolved https:// url, ` +
-          `got ${JSON.stringify(pointerOrUrl)} `
+    url = new URL(walletAddressUrl)
+    if (url.protocol !== 'https:') {
```

Comment:
```suggestion
    if (!ALLOWED_PROTOCOLS.includes(url.protocol)) {
```

Reply:
The wallet address should always be served under `https:`.

Reply:
I meant if in dev we want to use a localhost Rafiki.